make-it/EmMakefile

```diff
@@ -111,7 +111,7 @@ EM_EXPORTS = -s EXPORTED_FUNCTIONS="[\
     '_free' \
     ]"
 
-EM_RUNTIME_EXPORTS = -s EXTRA_EXPORTED_RUNTIME_METHODS="[\
```

Comment:
[EXTRA_EXPORTED_RUNTIME_METHODS was deprecated](https://github.com/emscripten-core/emscripten/blame/a8a0d8e79ed5d4aa640e18ae7497ef4c48e5beaa/ChangeLog.md#L129) in favor of `EXPORTED_RUNTIME_METHODS`.